mysql/mysql_suite.go

```diff
@@ -35,13 +35,16 @@ func (s *MysqlSuite) SetupSuite() {
 	_, err = s.DBConn.Exec("set session sql_mode='STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION';")
 	require.NoError(s.T(), err)
 
-	s.Migration, err = runMigration(s.DBName, s.MigrationLocationFolder, s.DSN)
+	s.Migration, err = runMigration(s.DBConn, s.MigrationLocationFolder)
+	require.NoError(s.T(), err)
+	err, _ = s.Migration.Up()
```

Comment:
Why should we `up` the migration on setupsuite?

Reply:
Aw sorry, my bad. I forgot to remove this. I add this just to make sure this function is work-well